routes/api.php

```diff
@@ -25,6 +25,10 @@
 Route::put('task/{id}', [\App\Http\Controllers\TaskDemoController::class, 'update']);
 Route::delete('task/{id}', [\App\Http\Controllers\TaskDemoController::class, 'delete']);
 
+// TaskCommentController
+
+Route::delete('delete/{id}', [TaskCommentController::class, 'delete']);
```

Comment:
Please let your route show that you are deleting a comment. "delete/{id}" is too general. Something like "delete-comment{id}" should suffice.

Reply:
Do you mean I should do another pull request to do that
, can u kindly merge my work, please